samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<ul><li>'Home' should be changed to <em>'home'</em></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>